docs/release-notes.md

```diff
@@ -1,157 +1,101 @@
+    Very quick and small release this week to deliver some new features and fixes!  Going forward, we'll be moving to a monthly cadence for Open AMT Cloud Toolkit with sub components releasing when they have features or fixes ready.  Our hope is that users of the Open AMT Cloud Toolkit will see the features they need delivered even faster.  In order to manage the overhead involved with this faster release cadence, we'll be moving to a monthly roll-up and release announcement starting at the end of November where we will summarize all of the releases during the previous month.  You can find the details of this release under [what's new](#whats-new) section.
 
     *Best wishes,*  
     *The Open AMT Cloud Toolkit Team*
 
 
 ## What's New?
 
-:material-star:** Community Contribution: Intel® AMT Alarm Clock Feature**
+:material-new-box:** Feature: AMT Maintenance commands **
 
-Both WSMAN-Messages and MPS now support setting both one-time and periodic wake-up alarms in AMT. To demonstrate the new API, the Sample Web UI offers a new control on the device page, allowing a user to set alarm clocks for a given device. Thank you Orin Eman from [Laplink](https://everywhere.laplink.com/) for this community contribution.
+Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
 
-:material-new-box:** Improvement: State Machine for Activation and Configuration**
+:material-star:** Customer Request: Unprovision AMT devices via MPS **
 
-In an effort to continuously improve Open AMT Cloud Toolkit, the team overhauled the activation and configuration flows inside of RPS. Utilizing [xstate](https://xstate.js.org/), we modularized each of the configuration actions so they can be called as needed by the state machine. This will add increased robustness to complex configuration flows as well as make it easier to add new configuration options in the future.
+We've had several customer requests to add this feature to MPS and this release we are delivering.  We've added an API end point to MPS that will send the unprovision command to AMT and will then remove that AMT device from the MPS database.  In order to reconnect that AMT device to MPS again, RPC and RPS will need to be used.
```

Comment:
API end point => API endpoint

Reply:
We've had several customer requests to add this feature to MPS and this release we are delivering. We've added an API endpoint (`DELETE /api/v1/amt/deactivate/{guid}`) to MPS that will send the unprovision command to a connected AMT Device and will then remove that AMT device from the MPS database. In order to reconnect that AMT device to MPS again, RPC and RPS will need to be used.